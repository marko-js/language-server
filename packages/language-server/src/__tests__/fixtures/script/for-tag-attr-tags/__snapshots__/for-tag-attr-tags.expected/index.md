## Hovers
### Ln 6, Col 8
```marko
  4 |     { foo: "c", bar: "d" },
  5 |   ]>
> 6 |     <@row>
    |        ^ (property) "@row": Marko.AttrTag<{
    cell?: Marko.AttrTag<{
        renderBody: Marko.Body;
    }>;
}> | undefined
  7 |     // ^?
  8 |       <@cell>${row.foo}</@cell>
  9 |       // ^?     ^?
```

### Ln 8, Col 10
```marko
   6 |     <@row>
   7 |     // ^?
>  8 |       <@cell>${row.foo}</@cell>
     |          ^ (property) "@cell": Marko.AttrTag<{
    renderBody: Marko.Body;
}> | undefined
   9 |       // ^?     ^?
  10 |       <@cell>${row.bar}</@cell>
  11 |     </@row>
```

### Ln 8, Col 17
```marko
   6 |     <@row>
   7 |     // ^?
>  8 |       <@cell>${row.foo}</@cell>
     |                 ^ (parameter) row: {
    foo: string;
    bar: string;
}
   9 |       // ^?     ^?
  10 |       <@cell>${row.bar}</@cell>
  11 |     </@row>
```

