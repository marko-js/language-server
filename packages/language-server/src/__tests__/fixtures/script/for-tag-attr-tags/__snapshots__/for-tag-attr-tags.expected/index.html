<div>
  <div>
    
    
      placeholder
      
      placeholder
    
  </div>
</div>